ethportal-peertest/src/events.rs

```diff
@@ -12,25 +12,22 @@ pub const STATE_NETWORK: &str = "state";
 pub const UTP_PROTOCOL: &str = "utp";
 
 pub struct PortalnetEvents {
-    pub overlay: Arc<OverlayProtocol>,
+    pub history_overlay: Arc<OverlayProtocol>,
+    pub state_overlay: Arc<OverlayProtocol>,
     pub protocol_receiver: mpsc::Receiver<Discv5Event>,
     pub utp_listener: UtpListener,
 }
 
 impl PortalnetEvents {
-    pub async fn new(overlay: Arc<OverlayProtocol>, utp_listener: UtpListener) -> Self {
-        let protocol_receiver = overlay
-            .discovery
-            .write()
-            .await
-            .discv5
-            .event_stream()
-            .await
-            .map_err(|e| e.to_string())
-            .unwrap();
-
+    pub async fn new(
+        history_overlay: Arc<OverlayProtocol>,
+        state_overlay: Arc<OverlayProtocol>,
+        protocol_receiver: mpsc::Receiver<Discv5Event>,
```

Comment:
Yeah, that's what I use throughout most of the PR, but here I assume I just left the name as is to minimize refactors. I can create a refactor PR to standardize this convention throughout the codebase so that it's not part of anything tied to the overlay protocol.